### PR TITLE
ntopng: 5.6 -> 6.0

### PR DESCRIPTION
**Pull request by ajs124**
## Description of changes
https://github.com/NixOS/nixpkgs/commit/f8c5ec05a81ad896b3179f7aba2e1ec3f759dc71

https://github.com/ntop/ntopng/releases/tag/6.0
- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) (or backporting [23.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md) and [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).
---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Review by bjornfor — APPROVED** [review]
LGTM. (Didn't test.)

File: pkgs/development/libraries/ndpi/default.nix [hunk 1]
```diff
@@ -12,20 +11,17 @@
 
 stdenv.mkDerivation rec {
```
Comment (gepbird) [inline pkgs/development/libraries/ndpi/default.nix:12]: Consider using the finalAttrs pattern.

File: pkgs/tools/networking/ntopng/default.nix [hunk 2]
```diff
@@ -26,13 +26,13 @@
 
 stdenv.mkDerivation rec {
```
Comment (gepbird) [inline pkgs/tools/networking/ntopng/default.nix:27]: Same here.

**Review by gepbird — CHANGES_REQUESTED** [review]
LGTM, just a few nits.

Result of `nixpkgs-review pr 284855` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages built:</summary>
  <ul>
    <li>ndpi</li>
    <li>ntopng</li>
  </ul>
</details>

**Comment by ajs124** [comment]
I didn't test upgrading from a previous version, but I did deploy this version and it worked.

Since I haven't deployed any older version on the system, I can't tell if anything relevant changed.

**Review by gepbird — APPROVED** [review]

**Pull request merged**